### PR TITLE
added a note on registration

### PR DESCRIPTION
**Pull request by bthirion**

### CODE REVIEW

File: _sections/intro.md [hunk 1]
```diff
@@ -16,4 +16,4 @@ In the 2020 Dev Days, we hope to **increase the awareness of Nilearn's developme
 We also hope to **highlight future directions** where we are especially excited to see **community-driven development** efforts.
 
 
-
+> the event does not require registration to listen
```
Comment (emdupre) [inline _sections/intro.md]: Light suggestion to specify that we would prefer if folks register for the sprint, but not the talks !

```suggestion
The talks for the Software Development and Scientific days will be freely available with public links shared on the event schedule.
We strongly encourage registration for the Code Sprint, which can be done [here](/dev-days-2020/sprint.html)
```

Comment (kchawla-pi) [reply _sections/intro.md:19]: I would recommend highlighting this point, in a big way. a lot of people seem to be asking about this.

**Comment by kchawla-pi** [comment]
I would recommend highlighting this point, in a big way. A lot of people seem to be asking about this.

+1 to Elizabeth's suggestion

**Comment by kchawla-pi** [comment]
If people need to register for chats and discussions during conferences, or can just do it then, we should also mention that.
Currently it is not highlighted that there will be live discussions and questioning available during the talks and that might be a draw for many.

**Pull request merged**